### PR TITLE
Add properties to kafka struct

### PR DESCRIPTION
**Pull request by lostick**

### CODE REVIEW

File: shipcat_definitions/src/region.rs [hunk 1]
```diff
@@ -119,6 +119,9 @@ pub struct KafkaConfig {
     ///
     /// These are injected in to the manifest.kafka struct if it's set.
     pub brokers: Vec<String>,
+
+    /// A mapping of kafka properties to environment variables (optional)
+    pub propertyEnvMapping: BTreeMap<String, String>,
```
Comment (clux) [inline shipcat_definitions/src/region.rs]: stick a `#[serde(default)]` on this please

Comment (lostick) [reply shipcat_definitions/src/region.rs]: well spotted

Comment (clux) [reply shipcat_definitions/src/region.rs]: actually can this be put under a kafka struct in `Region`?

**Comment by lostick** [comment]
@clux I've bumped `shipcat` to `0.93.0`

**Pull request merged**